### ISSUE TITLE
Bazel CI: Intellij Plugin Aspect broken with Bazel@HEAD

### ISSUE THREAD

**Issue by coeuvre** [bazelbuild/bazel#13548]
https://buildkite.com/bazel/bazel-at-head-plus-downstream/builds/2054#118c67db-feac-48b2-90d5-ed42876e43e9

```
JUnit4 Test Runner
.E
Time: 0.16
There was 1 failure:
1) testJavaProtoLibrary(com.google.idea.blaze.aspect.proto.jpl.JavaProtoLibraryTest)
expected to be empty
but was:
    [jar {
      relative_path: "aspect/testing/tests/src/com/google/idea/blaze/aspect/proto/jpl/libbar_proto-speed.jar"
      root_execution_path_fragment: "bazel-out/k8-fastbuild/bin"
      is_new_external_version: true
    }
    interface_jar {
      relative_path: "aspect/testing/tests/src/com/google/idea/blaze/aspect/proto/jpl/libbar_proto-speed-hjar.jar"
      root_execution_path_fragment: "bazel-out/k8-fastbuild/bin"
      is_new_external_version: true
    }
    source_jar {
      relative_path: "aspect/testing/tests/src/com/google/idea/blaze/aspect/proto/jpl/bar_proto-speed-src.jar"
      root_execution_path_fragment: "bazel-out/k8-fastbuild/bin"
      is_new_external_version: true
    }
    source_jars {
      relative_path: "aspect/testing/tests/src/com/google/idea/blaze/aspect/proto/jpl/bar_proto-speed-src.jar"
      root_execution_path_fragment: "bazel-out/k8-fastbuild/bin"
      is_new_external_version: true
    }
    ]
	at com.google.idea.blaze.aspect.proto.jpl.JavaProtoLibraryTest.testJavaProtoLibrary(JavaProtoLibraryTest.java:47)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
 
FAILURES!!!
Tests run: 1,  Failures: 1
```

Culprit Finder: https://github.com/bazelbuild/bazel/commit/f1b37b5d610728634d92ef4ebd4d1435c0c8605d

**Comment by coeuvre** [comment]
@kevin1e100 I am not familiar with this change but is this expected and the tests should be updated accordingly?

**Comment by meteorcloudy** [comment]
@kevin1e100 @comius Please take a look at this ;)

**Comment by kevin1e100** [comment]
+1. It seems to me the actual test output is strictly better than the expected, and looks WAI for me, so I'd prefer adjusting the test if that's possible

**Comment by philwo** [comment]
@mai93 IIUC seems like this should be just a little update on the plug-in side to fix a test, could you have a look?

**Comment by mai93** [comment]
Fixed after f1b37b5d610728634d92ef4ebd4d1435c0c8605d is rolled back in 260015df1524543d56b0dbff00958320cd58d166.
